gcloud/bigquery/table.py

```diff
@@ -693,6 +699,225 @@ def insert_data(self,
 
         return errors
 
+    @staticmethod
+    def _configure_job_metadata(metadata,  # pylint: disable=R0913
```

Comment:
1. Please use `disable=too-many-arguments`
1. Why don't we just raise the number of allowed arguments? (Recall the pain we went through to avoid sprinkling these throughout)